index.js

```diff
@@ -108,19 +108,19 @@ module.exports = () => {
 			task: async () => {
 				await execa('npm', [
 					'install',
-					'meow',
-					'ink',
+					'meow@9',
+					'ink@3',
```

Comment:
Since there's no 4.x version of Ink, do we need `@3` version range?

Reply:
For avoiding ESM-related issues in the future, we would better limit its version.